Libplanet.Standalone/Libplanet.Standalone.csproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Libplanet" Version="0.10.0-dev.20200911094212" />
```

Comment:
I guess we should bump it too.

Reply:
Thanks. I bumped it also, please check it again 🙏